src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -399,13 +400,31 @@ ErrorIfQueryNotSupported(Query *queryTree)
 		errorHint = filterHint;
 	}
 
-	if (queryTree->setOperations)
+	if (queryTree->setOperations && !subqueryPushdown)
```

Comment:
Per our discussion I am reverting these to be handled when we are doing union work